source/_templates/layout.html

```diff
@@ -1,7 +1,16 @@
 {% extends "!layout.html" %}
 
 {%- block htmltitle %}
+<!-- Jquery already required by bootstrap, and two uses of $(document), current document site throwing js errors without this -->
+  <script src=" https://cdn.jsdelivr.net/npm/jquery@3.7.1/dist/jquery.min.js "></script>
```

Comment:
In any case, let's not depend on a third-party CDN, these tend to break in the long term.

Reply:
Resolved, also same action applied to the lucide icons script